examples/http/crestron_tsw.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * Domotz Custom Driver 
+ * Name: Crestron TSW
+ * Description: This script its designed to monitor the device's status and synchronization with external calendars
+ *   
+ * Communication protocol is HTTPS
+ * 
+ * Tested with Crestron TSW-760 version: 3.002.1025 
+ *
+ * Custom Driver Variables:
+ *      - CalendarSyncStatus: Monitors the synchronization status of the device's calendar.
+ *      - ConnectionStatus: Tracks the device's connection status.
+ *      - ConnectionStatusMessage: Provides a message related to the connection status.
+ *      - ExchangeRegistrationStatus: Monitors the registration status of the device with Microsoft Exchange.
+ *
+ **/
+
+/**
+ * @param {*} init object initialisation
+ * @param {*} object object to clone
+ * @returns cloned object
+ */
+function clone(init, object) {
```

Comment:
I don't see much need for this function.

It would be better to just define the two dictionaries in the respective functions (login, get device)